models/brats_mri_generative_diffusion/configs/inference.json

```diff
@@ -63,12 +65,20 @@
         ],
         "num_res_blocks": 2
     },
+    "trt_args": {
```

Comment:
I see every bundle in this PR contain the same `trt_args`, can we set some default values for `trt_args` in the `trt_wrap` function? Then no need to write this part for every bundle.

Thanks.

Reply:
Yes I was thinking about this, too. Let me take another stab at this.

Reply:
This is implemented now.